### ISSUE TITLE
Suddenly unable to download

### ISSUE THREAD

**Issue by gitupwithit** [JannisX11/blockbench#89]
![screenshot from 2017-11-20 12-21-41](https://user-images.githubusercontent.com/32688796/33031993-de649cc6-cded-11e7-9d30-fb775575261c.png)

Hi, the download button appears to be frozen ? I can export ok, but no download. I can click but nothing happens, even if I wait a few minutes it doesn't download.

(Using web version on Ubuntu 16.10 and Chrome)

Thanks!

**Comment by Toastrackenigma** [comment]
This happened to me just now too. Looking in the Chrome Dev Tools, it throws an error:

> Uncaught TypeError: Cannot read property 'join' of undefined

when the `saveFileBlock()` function is called. Upon further inspection, this is because `display.fixed` (the display view for Item Frames) did not have a `.scale` property - as I had not touched the scale sliders for that display property, and then the code was trying to `.join` on that property, thus throwing the error.

What is interesting is that I had other display scale sliders set at 1, 1 and 1, and in a new project where I had not touched any display settings download worked just fine. Therefore I am assuming that something has to happen in the display settings for this to be an issue? Maybe moving the sliders from 1, 1, 1 and then dragging them back?

I fixed my issue by moving the sliders to some other values (I set mine to 0.75, 0.75, 0.75, but it shouldn't really matter) and then the model downloaded fine.

I was going to add a pull request fixing this in the code, but it seems like it has already been fixed here on GitHub. So I guess we just have to wait until that version is pushed to the website.
